components/Masthead/Masthead.tsx

```diff
@@ -63,6 +64,7 @@ export const Masthead = ({ isLight, className }: MastheadProps) => {
       animate={{ y: isVisible ? 0 : -slideDistance, opacity: isVisible ? 1 : 0 }}
       transition={{ duration: 0.2, delay: !isAtTop && isScrollingBack ? 0.4 : 0, ease: 'easeInOut' }}
     >
+      <Skiplink />
```

Comment:
I move the skip link within the `header` landmark

Reply:
![Screenshot 2023-10-20 at 11 35 51 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/2400be1e-2fb1-46f1-b423-7668a75d2f3d)